tests/test_runner/test_utils.py

```diff
@@ -76,8 +76,7 @@ def __init__(self):
                                          "--noconsole"
                                         ],
                                         universal_newlines=True,
-                                        stdout=subprocess.PIPE,
-                                        stderr=subprocess.PIPE)
+                                        stdout=subprocess.PIPE)
```

Comment:
For DobbyDaemon all logs are printed into stderr, this is why it was added. For now it would work with only stderr=subprocess.PIPE. I left both as when it get fixed "normal" output should be in stdout so we will not need to change it. With your proposed change some tests can not shown proper messages.
Please revert this line as it was before.

Reply:
Good catch! That was not done on purpose, and slipped through when I reviewed my own changes. Thanks!